metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -721,15 +739,12 @@ def merge_column(
         original_dict: dict, refinement_dict: dict, key: str
     ) -> List[dict]:
         """
-        Check refinement parameter override at https://help.looker.com/hc/en-us/articles/4419773929107-LookML-refinements
+        Merge a dimension/measure/other column with one from a refinement.
+        This follows the process documented at https://help.looker.com/hc/en-us/articles/4419773929107-LookML-refinements
         """
         merge_column: List[dict] = []
-        original_value: Any = (
-            original_dict.get(key) if original_dict.get(key) is not None else []
-        )
-        refine_value: Any = (
-            refinement_dict.get(key) if refinement_dict.get(key) is not None else []
-        )
+        original_value: Any = original_dict.get(key, [])
```

Comment:
please improve the type annotations and avoid using Any
```suggestion
        original_value: List[dict] = original_dict.get(key, [])
```